automation/roles/pgbouncer/tasks/main.yml

```
@@ -100,22 +100,21 @@
         group: root
         state: link
       when: not pgbouncer_bin.stat.exists
-  when: ansible_os_family == "RedHat" and
-        postgresql_packages|join(" ") is search("postgrespro")
+  when: ansible_os_family == "RedHat" and postgresql_packages|join(" ") is search("postgrespro")
```

```suggestion
  when:
    - ansible_os_family == "RedHat"
    - postgresql_packages|join(" ") is search("postgrespro")
```
we can use a list instead of `and`